Ihor/Netflix/src/MovieDetails/MovieInfo/MovieInfo.tsx

```diff
@@ -1,15 +1,21 @@
 import React from "react";
 import IMovie from "../../Interfaces/IMovie";
 import "./MovieInfo.css";
+import notfound from "../../img/notfound.png";
 
 interface IMovieProps {
   movie: IMovie;
 }
 
 const MovieInfo = ({ movie }: IMovieProps) => {
+
+  const handleImageError = (event: React.SyntheticEvent<HTMLImageElement>) => {
+    const target = event.target as HTMLImageElement;
+    target.src = notfound;
+  };
   return (
     <div className="film_info">
-      <img className="m_img" src={movie.poster_path} alt={movie.title} />
+      <img src={movie.poster_path} alt={movie.title} className="m_img" onError={(event) => handleImageError(event)}/>
```

Comment:
You can make a reusable Image component where you will make this check, so you won't need to add handleImageError everytime.